src/icon-manifest/languages.ts

```diff
@@ -58,112 +58,112 @@ export const languages: ILanguageCollection = {
   glsl: { ids: 'glsl', defaultExtension: 'glsl' },
   go: { ids: 'go', defaultExtension: 'go' },
   godot: { ids: 'gdscript', defaultExtension: 'gd' },
-  graphql: { ids: ['graphql'], defaultExtension: 'gql' },
-  graphviz: { ids: ['dot'], defaultExtension: 'gv' },
+  graphql: { ids: 'graphql', defaultExtension: 'gql' },
+  graphviz: { ids: 'dot', defaultExtension: 'gv' },
   groovy: { ids: 'groovy', defaultExtension: 'groovy' },
-  haml: { ids: ['haml'], defaultExtension: 'haml' },
+  haml: { ids: 'haml', defaultExtension: 'haml' },
   handlebars: { ids: 'handlebars', defaultExtension: 'hbs' },
   harbour: { ids: 'harbour', defaultExtension: 'prg' },
-  haskell: { ids: ['haskell'], defaultExtension: 'hs' },
+  haskell: { ids: 'haskell', defaultExtension: 'hs' },
   haxe: { ids: ['haxe', 'hxml', 'Haxe AST dump'], defaultExtension: 'haxe' },
   hlsl: { ids: 'hlsl', defaultExtension: 'hlsl' },
   html: { ids: 'html', defaultExtension: 'html' },
   http: { ids: 'http', defaultExtension: 'http' },
   ini: { ids: 'ini', defaultExtension: 'ini' },
-  latex: { ids: ['latex'], defaultExtension: 'tex' },
+  latex: { ids: 'latex', defaultExtension: 'tex' },
   java: { ids: 'java', defaultExtension: 'java' },
   javascript: { ids: 'javascript', defaultExtension: 'js' },
   javascriptreact: { ids: 'javascriptreact', defaultExtension: 'jsx' },
-  jinja: { ids: ['jinja'], defaultExtension: 'jinja' },
+  jinja: { ids: 'jinja', defaultExtension: 'jinja' },
   json: { ids: 'json', defaultExtension: 'json' },
   json5: { ids: 'json5', defaultExtension: 'json5' },
   julia: { ids: ['julia', 'juliamarkdown'], defaultExtension: 'jl' },
-  kos: { ids: ['kos'], defaultExtension: 'ks' },
-  kotlin: { ids: ['kotlin'], defaultExtension: 'kt' },
+  kos: { ids: 'kos', defaultExtension: 'ks' },
+  kotlin: { ids: 'kotlin', defaultExtension: 'kt' },
   less: { ids: 'less', defaultExtension: 'less' },
-  lisp: { ids: ['lisp'], defaultExtension: 'lisp' },
+  lisp: { ids: 'lisp', defaultExtension: 'lisp' },
   literatehaskell: { ids: ['literate haskell'], defaultExtension: 'lhs' },
```

Comment:
I guess this one can also be simplified.

Reply:
Yeah. Just leave it for today. I need to fix the code coverage aswell.